### DISCUSSION TITLE
Is this conda compatible? (question is more neuanced, please read)

### DISCUSSION THREAD

**Discussion by Yasand123** [Comfy-Org/ComfyUI-Manager#624] [Q&A] [answered]
I know it works within a conda environment, but will it use conda to manage python depends or does it only use pip? This is important because if it uses conda it would utilize the caching system to prevent unnecessary re-downloading of packages.

Thanks.

**Comment by ltdrdata** [comment]
I'll update it.

**Comment by ltdrdata** [comment, marked as answer]
After a simple investigation, it appears that when pip is installed using `conda install pip`, the cache between conda and pip is shared, so packages already installed by conda are not reinstalled by pip.

Do you need anything more than this?

Furthermore, since there are often packages that cannot be installed with `conda install`, it seems that there will be cases where one must attempt with conda and, if that fails, retry with pip... This also seems to have potential problems.

**Reply by Yasand123** [reply]
Yeah, it's possible that conda will not have a pip package that a plugin might need, which might introduce unnecessary complexity. If it's as you say that pip attempts to use cached packages first that were downloaded by conda, then I don't think adding explicit conda support for the plugin manager is really beneficial especially considering the potential downsides. So I guess this question is answered.

Thank you for your time and confirmation. Have a great day.
